### PR TITLE
resolves GPS-219

### PR DESCRIPTION
**Pull request by klpittman**
- Sets aria-expanded to true/false when the auto-complete results are shown/hidden.
- Adds aria description to the input for screen-reader users. This text could be even more descriptive once we decide exactly how to interact with the widget via keyboard.
- adds the for="search" attribute to the <label>
- Adds the following to the input so screen-readers have more info about what it is and how it works: `aria-describedby="search-description" aria-owns="suggestions" aria-haspopup="true" aria-expanded="false" aria-autocomplete="both"`
- Adds `aria-live="polite" role="listbox"` to the div where the suggestions are shown. 
  - Note: This list still needs to add attributes to the list options, but i'm not sure where to do that.
- Adds `role="alert"` to the <div> containing the error warning text. This allows the screen-reader to interrupt the user with a warning when no results will be found.

### CODE REVIEW

File: pivot/templates/major-gpa.html [hunk 1]
```diff
@@ -10,48 +10,52 @@ <h1 class="page-title">View Major GPAs</h1>
...
+							            <ul class="dropdown-menu" aria-labelledby="dropdownMenu">
+							            </ul>
+							        </div>
+							    </div>
+							</div>
+							<div class="col-xs-12 col-sm-5"><!-- TODO ADD BOOTSTRAP CLASS TO REMOVE LEFT & RIGHT GUTTERS -->
+							    <div class="form-group">
+							        <!-- major search -->
+							        <label class="sr-only" for="search">Major</label>
+									<div class="sr-only" id="search-description">Enter the name of a major. Matches results will be shown below as you type.</div>
```
Comment (hlee2) [inline pivot/templates/major-gpa.html]: why not using the <label>?

File: pivot/templates/major-gpa.html [hunk 2]
```diff
@@ -10,48 +10,52 @@ <h1 class="page-title">View Major GPAs</h1>
...
+							        <input id="search" class="form-control" type="text" name="searchQuery" value="" size="30" tabindex="0" data-save-value="true" data-role="Control" data-control_for="search" placeholder="e.g. Computer Science, Art History" autocomplete="off" aria-describedby="search-description" aria-owns="suggestions" aria-haspopup="true" aria-expanded="false" aria-autocomplete="both" aria-activedescendant="" />
```
Comment (hlee2) [inline pivot/templates/major-gpa.html]: I am not sure why the old search input field style is replacing the current.

File: pivot/templates/major-gpa.html (hunk 1 above)
Comment (klpittman) [reply pivot/templates/major-gpa.html]: I don't see a "?" anywhere on the screen. What "?" are you talking about?

Comment (klpittman) [reply pivot/templates/major-gpa.html]: I am using the `<label>`, see above. 

The search-description just adds extra information for screen readers and is referenced by the search input's attribute "describedby". When the user activates the search it should be read and provide more information about how to use it or what it is.

**Pull request merged**